pyproject.toml

```diff
@@ -17,13 +17,15 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.8"
-requests = "^2.25"
-sceptre = ">=4.0.1"
```

Comment:
Should `sceptre` have been moved to the `dev` group? It seems to me that this resolver can't function without `sceptre` installed.

Reply:
We make the assumption that users should know that they need to install Sceptre core for a plugin to work.  With that assumption Sceptre is only really needed for developing this plugin.  Also we do not want to package Sceptre core into this plugin package.